service/management-ui/page/courses/add.html

```diff
@@ -0,0 +1,40 @@
+<Page title="Add course">
+    <Layout page='add-course'>
```

Comment:
Why do we keep passing in a page parameter? The `Layout.html` component doesn't seem to refer to the variable?